source/_integrations/binary_sensor.template.markdown

```diff
@@ -294,6 +294,31 @@ binary_sensor:
 
 {% endraw %}
 
+### Rate limiting updates
+
+When there are entities present in the template, the template will be re-rendered when one of the entities changes states.
+
+When `states` is used in a template by itself to iterate all states on the system, the template is re-rendered each
+time any state changed event happens if any part of the state is accessed. When merely counting states, the template
```

Comment:
I think this description does not match the last updates to the core PR?

Also, it does not mention how to disable the rate limiting.

Reply:
I tweaked it here https://github.com/home-assistant/home-assistant.io/pull/14776